sphinxcontrib/drawio/__init__.py

```diff
@@ -230,16 +225,26 @@ def drawio_export(builder: Builder, options: dict, in_filename: str,
 
 def on_config_inited(app: Sphinx, config: Config) -> None:
     if is_headless(config):
-        process = subprocess.Popen(["Xvfb", ":{}".format(X_DISPLAY_NUMBER), "-screen", "0", "1280x768x16"],
-                                   stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-        config.xvfb_pid = process.pid
-
-        if process.poll() is not None:
-            raise OSError("Failed to start Xvfb process"
-                          "\n[stdout]\n{}\n[stderr]{}".format(*process.communicate()))
-
+        logger.info("running in headless mode, starting Xvfb")
+        with TemporaryFile() as fp:
+            fd = fp.fileno()
+            xvfb = Popen(["Xvfb", "-displayfd", str(fd), "-screen",
+                          "0", "1280x768x16"], pass_fds=(fd, ),
```

Comment:
any reason why this display size was chosen specifically?

Reply:
I have no idea. Do [you](https://github.com/Modelmat/sphinxcontrib-drawio/commit/8d5aa4b799fef4c41acb2ec14751408eb757a2bf#diff-74ea612f243ff0b44c6c2000a0e8d6ede7599341ce76fdeff8a5ee9aafc1a004R200)? 😉

Reply:
Haha :)
I guess not..